test/unit/watchable-function-logic/misc.spec.ts

```diff
@@ -30,4 +30,10 @@ describe('WatchableFunctionLogic: Miscellaneous', () => {
       fake.receiveBoolean.should.always.have.been.calledOnceWith(true);
     }).to.throw('always flag is not supported for method calledOnceWith');
   });
+
+  it('should throw when expecting a delegatedFrom that did not happen', async () => {
+    expect(() => {
+      fake.receiveBoolean.should.to.be.delegatedFrom(fake.address);
```

Comment:
is the `.to.be` necessary or can we just use `should.be`?

Reply:
pushed `should.be`